### PR TITLE
[CFM-1561] Implement feedback on invitation modules.

### PR DESCRIPTION
**Pull request by ybaras**
[CFM-1561](https://issuetracker.amplexor.com/jira/browse/CFM-1561)

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_users/c4m_og_invite_users.module [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * other existing users to the group.
  */
 function c4m_og_invite_users_menu_alter(&$items) {
-  $items['group/%/%/admin/people/invite-users']['access callback'] = 'c4m_og_invite_users_access';
+  $items['group/%/%/admin/people/invite-users']['access callback'] = 'c4m_og_invite_users_deny_access';
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_users/c4m_og_invite_users.module]: Add a comment for this block

**Comment by ybaras** [comment]
Implement the unified workflow on invitation for visitors

title

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.module [hunk 2]
```diff
@@ -309,7 +309,10 @@ function c4m_user_profile_form_user_register_form_alter(
   &$form_state,
   $form_id
 ) {
-  drupal_set_title(t('Registration'));
+
+  if (empty($_SESSION['c4m_og_invite_visitors_gid_join'])) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.module]: comment

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_visitors/c4m_og_invite_visitors.module [hunk 3]
```diff
@@ -523,6 +523,16 @@ function c4m_og_invite_visitors_form_user_register_form_alter(&$form, &$form_sta
     return;
   }
 
+  $wrapper = entity_metadata_wrapper('node', $_SESSION['c4m_og_invite_visitors_gid_join']);
+  $variables = array(
+    'group_title' => $wrapper->label(),
+    'login_link' => l(t('Log in.'), 'user/login'),
+  );
+  drupal_set_title(
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_visitors/c4m_og_invite_visitors.module]: let's keep this one in one line

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_visitors/c4m_og_invite_visitors.module [hunk 4]
```diff
@@ -242,7 +242,12 @@ function _c4m_og_invite_visitors_new_users_form($form, &$form_state, $gid) {
     '#type' => 'value',
     '#value' => $group->nid,
   );
-
+  $form['manage_group_people'] = array(
+    '#type' => 'link',
+    '#href' => str_replace('/invite-visitors', '', $_GET['q']),
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_visitors/c4m_og_invite_visitors.module]: please explain

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_visitors/c4m_og_invite_visitors.module [hunk 5]
```diff
@@ -361,7 +366,16 @@ function c4m_og_invite_visitors_determine_user_action($invitee_email, $gid) {
 
   // Do not invite a user who is already a member of the group.
   if ($og_membership = og_get_membership('node', $gid, 'user', $account->uid)) {
-    drupal_set_message(t('%user is already a member of this group.', array('%user' => format_username($account))), 'error');
+    // Member is blocked, display a link to the manage people page.
+    if ($og_membership->state == 3) {
+      drupal_set_message(t('%user is a blocked member of this group. <a href="@people-page">Manage the group members.</a>', array(
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_visitors/c4m_og_invite_visitors.module]: we do not add html elements inside a `t()` function

File: project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_visitors/c4m_og_invite_visitors.module [hunk 6]
```diff
@@ -367,10 +368,11 @@ function c4m_og_invite_visitors_determine_user_action($invitee_email, $gid) {
...
   if ($og_membership = og_get_membership('node', $gid, 'user', $account->uid)) {
     // Member is blocked, display a link to the manage people page.
-    if ($og_membership->state == 3) {
-      drupal_set_message(t('%user is a blocked member of this group. <a href="@people-page">Manage the group members.</a>', array(
+    if ($og_membership->state == OG_STATE_BLOCKED) {
+      $link = l(t('Manage the group members.'), 'group/node/' . $og_membership->gid . '/admin/people');
+      drupal_set_message(t('%user is a blocked member of this group. !people-page', array(
         '%user' => format_username($account),
-        '@people-page' => url('group/node/' . $og_membership->gid . '/admin/people'),
+        '!people-page' => $link,
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_visitors/c4m_og_invite_visitors.module]: `link` should be part of the variable

**Pull request merged**